src/App.js

```diff
@@ -40,23 +59,56 @@ function App() {
 
   return (
     <Container>
-      {answers && (
-        <PlayPage
-          showAnswer={isAnswerVisible}
-          answers={newAnswers || answers}
-          onPlay={handlePlay}
-          onAnswer={handleAnswer}
-          playing={isPlaying}
-          duration={duration}
-          onChange={handleVolume}
-        />
-      )}
+      <Route path="/playpage">
+        <Navigation onBack={handleBack} />
+      </Route>
+      <Switch>
+        <Route exact path="/">
+          <StartPage
+            playlists={playlists}
+            onMark={handleMark}
+            onGame={handleGame}
+            selectedPlaylist={selectedPlaylist}
+          />
+        </Route>
+        <Route path="/playpage">
+          {answers && (
+            <PlayPage
+              showAnswer={isAnswerVisible}
+              answers={newAnswers || answers}
+              onPlay={handlePlay}
+              onAnswer={handleAnswer}
+              isPlaying={isPlaying}
+              duration={duration}
+              onChange={handleVolume}
+              isLoaded={isLoaded}
+            />
+          )}
+        </Route>
+      </Switch>
```

Comment:
If the `Navigation` is only shown on `/playpage`, maybe put it inside the `Route` below where the `PlayPage` component is!?